share/p2p/peers/manager_test.go

```diff
@@ -217,6 +216,8 @@ func TestIntegration(t *testing.T) {
 		require.NoError(t, bnPubSub.Start(ctx))
 		require.NoError(t, fnPubSub.Start(ctx))
 
+		require.NoError(t, nw.ConnectAllButSelf())
+
```

Comment:
I've sneaked in test fix here. 

Thank you @vgonkivs for figuring out the root cause and how it could be fixed